LayoutTests/fast/dom/dynamic-image-with-lazy-loading-leak.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="../../resources/js-test.js"></script>
+<div style="height: 2000px"></div>
+<div id="below-viewport-img-container" style="height: 1000px"></div>    
+<script>
+description("Tests that lazy image loading doesn't cause leaks when 'img' element is created and removed dynamically.");
+jsTestIsAsync = true;
+
+img = null;
+const image_path = 'image.png';
+imgElementId = null;
+
+function runTest()
+{
+    setTimeout(() => {
+        gc();
+        if (!internals.isElementAlive(imgElementId)) {
+            testPassed("The img element didn't leak.");
+        } else {
+            testFailed("The img element leaked.");
+        }
+        finishJSTest();
+    }, 10);
+}
+
+onload = () => {
+    imgElementId = internals.elementIdentifier(document.getElementById("lazy-loaded-img"));
+    shouldBeTrue("internals.isElementAlive(imgElementId)");
+    document.getElementById('below-viewport-img-container').removeChild(img);
+    img = null;
+    setTimeout(runTest, 0);
+};
+
+img = new Image();
```

Comment:
Because of the way our GC works, you should create several images (at least 10) and then verify that at least one gets GC'd. Allocating one and expecting it to get GC'd will likely lead to flakiness.